scripts/compactor_runner.py

```diff
@@ -235,6 +237,8 @@ def _run_corfu_compactor(self):
         with open(self._config.configPath, "r") as config:
             compactor_config = yaml.load(config)
         corfu_paths = compactor_config["CorfuPaths"]
+        # If this script is started by root user, ensure corfu user has permissions to ALL log files
+        os.system("chown corfu:corfu " + corfu_paths["CompactorLogfile"] + "*")
```

Comment:
Is the * required here? We don't need the rolled over logs to be accessible by corfu user.

Reply:
the rollover logic is executed by the log4j process writing these files, so when it finds that the max rollover files are reached it will try to delete the oldest one and fail if the permissions are not accessible right?

Reply:
no when the rollover happens it's always root. So we just need to have the corfu-compactor-audit.log to be corfu:corfu